tools/build/container_push.bzl

```diff
@@ -0,0 +1,16 @@
+load("@io_bazel_rules_docker//container:container.bzl", lib_push = "container_push")
+
+tags_to_push = ["latest", "beta"]
```

Comment:
Can you please also add "release" here
because I need it for #332

Reply:
Will do!